program/src/state.rs

```diff
@@ -75,18 +131,18 @@ pub struct ProgramSigner {
 }
 
 impl ProgramSigner {
-    pub fn address() -> Pubkey {
+    pub fn pubkey() -> Pubkey {
         pubkey!("4fDQAj27ahBfXw3ZQumg5gJrMRUCzPUW6RxrRPFMC8Av")
```

Comment:
It generates `Pubkey::new_from_array([])` so would be better to make this a constant instead of calling the function each time it's needed. I'll fix that.